api/routes.ts

```diff
@@ -9,7 +9,7 @@ import payments from './payments'
 import userDetailsRouter from './user'
 import getUserList from './userList'
 
-process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '1'
```

Comment:
We don't need to explicitly set this, I'd suggest we set it back to '0' but comment out that line

Reply:
lets test and assess the impact in AAT first plz

Reply:
Switching it back to 0 is a bad idea. It is a security issue.